piccolo_api/jwt_auth/endpoints.py

```diff
@@ -37,21 +32,17 @@ async def post(self, request: Request) -> JSONResponse:
         if not user_id:
             raise HTTPException(status_code=401, detail="Login failed")
 
-        expiry = datetime.now() + self._expiry
-
-        payload = jwt.encode({"user_id": user_id, "exp": expiry}, self._secret)
```

Comment:
We might have keep the `exp` value to the payload. I think this is how PyJWT works out if it has expired or not.

https://github.com/jpadilla/pyjwt/blob/5fe7f2b28ffcd27a3e520be750858944889112ef/jwt/api_jwt.py#L144-L145

Reply:
@dantownsend I think those are the claim names JWT generates itself. not our expiry in payload.
https://pyjwt.readthedocs.io/en/stable/usage.html#registered-claim-names

Reply:
@aminalaee  I can't see any other way of setting the expiry for the token. I don't think PyJWT sets it for us - at least I couldn't see anything in the codebase, but could be mistaken.

Reply:
@dantownsend You're right. It's the next paragraph in the docs. I'll restore that.